src/Components/Patient/PatientRegister.tsx

```diff
@@ -745,7 +748,9 @@ export const PatientRegister = (props: PatientRegisterProps) => {
     const data = {
       abha_number: state.form.abha_number,
       phone_number: parsePhoneNumber(formData.phone_number),
-      emergency_phone_number: parsePhoneNumber(formData.emergency_phone_number),
+      emergency_phone_number: isEmergencyNumberEnabled
+        ? parsePhoneNumber(formData.phone_number)
+        : parsePhoneNumber(formData.emergency_phone_number),
```

Comment:
Avoid changing this here.
Let's keep the formData / form state the source of truth.